src/libraries/System.Runtime.Intrinsics/ref/System.Runtime.Intrinsics.cs

```diff
@@ -5766,7 +5766,15 @@ internal Arm64() { }
         // public static void Scatter16BitNarrowing(System.Numerics.Vector<int> mask, System.Numerics.Vector<uint> addresses, System.Numerics.Vector<int> data) { throw null; }
         public static void Scatter16BitNarrowing(System.Numerics.Vector<long> mask, System.Numerics.Vector<ulong> addresses, System.Numerics.Vector<long> data) { throw null; }
         // public static void Scatter16BitNarrowing(System.Numerics.Vector<uint> mask, System.Numerics.Vector<uint> addresses, System.Numerics.Vector<uint> data) { throw null; }
```

Comment:
Could you please remind me why the int/unit variants of the `ScatterNarrowing` APIs are unimplemented for now?